<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="O3020MO0" ignored="0" name="Suite de tests" tests="4" failures="1" timestamp="18 dÃ©c. 2019 16:09:35 GMT" time="0.164" errors="0">
  <testcase name="test_B_1" time="0.007" classname="acial.selenium.exercices.TestB"/>
  <testcase name="test_B_2" time="0.011" classname="acial.selenium.exercices.TestB"/>
  <testcase name="test_A_2" time="0.005" classname="acial.selenium.exercices.TestA"/>
  <testcase name="test_A_1" time="0.012" classname="acial.selenium.exercices.TestA">
    <failure type="java.lang.AssertionError" message="expected [2] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [1]
at acial.selenium.exercices.TestA.test_A_1(TestA.java:23)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test_A_1 -->
</testsuite> <!-- Suite de tests -->
